src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+spring.jpa.hibernate.ddl-auto=none
```

Comment:
Let's use the YAML file instead, and remove the password from the file. Secrets like passwords should never be in plaintext, especially not for an open source project.

It's probably not a big deal in this case though, since it's not deployed anywhere, and we'll use a more secure password anyway if we do ship it at some point.